sec2_the_ruby_programming_language/2.12_final_ruby_project/4_mixins/crud_module.rb

```diff
@@ -0,0 +1,32 @@
+module Crud
```

Comment:
Why am I still seeing this file here? I thought this was reviewed previously?

Reply:
The file that was reviewed previously is the crud_module.rb in the 3_create_module folder which is merged in the [Section 2: Final Project - module](https://github.com/ziqiang2344/Learn_Ruby_on_Rails/pull/15) PR.

The difference between this file and the file mentioned above is that `self` keyword is added in front of the functions name but this one doesn't need it because the student.rb has included the Crud module.